cumulusci/tasks/robotframework/perf_listener.py

```diff
@@ -0,0 +1,37 @@
+from unittest import mock
+
+from robot.result.model import TestCase
+
+
+class PerfListener:
+    ROBOT_LISTENER_API_VERSION = 3
+    ROBOT_LIBRARY_SCOPE = "TEST CASE"
+    TEST_ELAPSED_TIMES = {}
+
+    def __init__(self):
+        self.ROBOT_LIBRARY_LISTENER = self
+
+    def start_test(self, data, result):
+        self.test_id = result.id
+
+    def end_test(self, data, result):
+        if self.test_id in self.TEST_ELAPSED_TIMES:
+            result.tags.add(("specified_elapsed_time",))
```

Comment:
@boakley Can you link to the docs for tagstoexclude ? It didn't pop up easily with Google.

Reply:
[Including and excluding tag statistics](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#including-and-excluding-tag-statistics) in the robot framework user guide.

Reply:
Aha, tagstatexclude = Tag Stat Exclude . I thought it was a misspelling of tags to exclude. You almost have to read it right-to-left because left-to-right it looks like"tags t..." or "tag state ..." !